Switch to pytest instead of nose

walro opened the pull request:
Close #43

walro committed:
Switch to pytest from nose

Nose is not maintained anymore.

Close #43

walro committed:
Removed unused main method

We always run via pytest so there's no need for this.

walro committed:
Remove unused import

walro committed:
Remove extra new-line

roback commented on Makefile:
```diff
@@ -10,4 +10,4 @@ localdeps:
 
 test:
-	nosetests
+	pytest --disable-pytest-warnings
```
Seems like `--disable-pytest-warnings` is just an alias to `--disable-warnings` at least according to the output from `pytest --help`. `--disable-warnings` is also the one mentioned in [the documentation](https://docs.pytest.org/en/latest/warnings.html#disabling-warnings-summary), so I think we can switch to that one instead.

walro replied:
7364208

roback reviewed (approved):
Just one small comment, other than that it LGTM. Ran it locally and it works great.

walro committed:
Use shorter form for disabling warnings

https://github.com/twingly/twingly-search-api-python/pull/47#discussion_r269018914
